systemtest/tests/integration/test.sh

```diff
@@ -59,6 +59,21 @@ if ! command -v bootc >/dev/null || bootc status | grep -q 'type: null'; then
   echo "info: running in bootc/image-mode"
 fi
 
+# Install the Echo worker for downstream tests.
+# Note: The Echo worker is not currently built  in yggdrasil,
+# but it is available in upstream COPR builds as part of the packaged distribution.
+if [ ! -x /usr/libexec/yggdrasil/echo ]; then
+  mkdir /usr/libexec/yggdrasil
+  go install github.com/redhatinsights/yggdrasil/worker/echo@latest
+  mv ~/go/bin/echo /usr/libexec/yggdrasil/echo
+
+  yggctl generate worker-data --name echo --program /usr/libexec/yggdrasil/echo --user yggdrasil --output dbusfile_worker -i
+  # currently there is a typo in service file generated by yggctl, adding a workaround
+  # https://github.com/RedHatInsights/yggdrasil/pull/343
+  sed -i 's/SystemdService=.*\.Service/SystemdService=com.redhat.Yggdrasil1.Worker1.echo.service/' \
+  /usr/local/share/dbus-1/system-services/com.redhat.Yggdrasil1.Worker1.echo.service
```

Comment:
Do we still need the workaround? 
[PR#343](https://github.com/RedHatInsights/yggdrasil/pull/343) is merged now.